api-gateway/src/routes/matchingRoutes.ts

```diff
@@ -0,0 +1,25 @@
+import express from 'express';
+import { createProxyMiddleware, fixRequestBody } from 'http-proxy-middleware';
+import config from '../config';
+import logger from '../utils/logger';
+import { authMiddleware } from '../middleware/authMiddleware';
+
+const router = express.Router();
+
+const matchingServiceProxy = createProxyMiddleware({
+  target: config.services.matching, // Matching Service URL
+  changeOrigin: true,
+  ws: true, // Enable WebSocket proxying
+  pathRewrite: {
+    '^/(.*)': '/$1', // front end connects to matching service via /ws only
```

Comment:
might want to still add the `api/v1/` here incase we want to do a health ping? Currently `match-svc` only has 1 route for health check